modules/custom/az_marketing_cloud/az_marketing_cloud.links.task.yml

```
@@ -0,0 +1,6 @@
+az_news_marketing_cloud.import:
```

Can we take news out of this identifier?
```suggestion
az_marketing_cloud.import:
```